modules/qna/src/views/full/Item/index.tsx

```diff
@@ -27,79 +27,56 @@ const Item: FC<Props> = props => {
     return null
   }
 
-  const { id, item, contentLang } = props
-
+  const { id, item, contentLang, last, isLite } = props
   const questions = item.questions[contentLang] || []
   const answers = item.answers[contentLang] || []
   const missingTranslations = !questions.length || (item.action !== ACTIONS.REDIRECT && !answers.length)
 
   return (
-    <div className={cx(style.qnaItem, style.well)} key={id}>
-      <div className={style.itemContainer} role="entry">
+    <div role="entry" className={cx(style.questionTableRow, { [style.last]: last })} key={id}>
+      <div className={cx(style.questionTableCell, style.question)}  onClick={() => props.onEditItem(id)}>
         {missingTranslations && (
-          <div className={style.itemQuestions}>
-            <a className={style.firstQuestionTitle} onClick={() => props.onEditItem(id)}>
-              <Tooltip content="Missing translation">
-                <Icon icon="warning-sign" intent={Intent.DANGER} />
-              </Tooltip>
-              &nbsp;
-              {id
-                .split('_')
-                .slice(1)
-                .join(' ')}{' '}
-              &nbsp;
-            </a>
-          </div>
+          <Fragment>
+            <Tooltip content="Missing translation">
+              <Icon icon="warning-sign" intent={Intent.DANGER} />
+            </Tooltip>
+            &nbsp;
+            {id
```

Comment:
I know you didn't come up with this but is this necessary ?

Reply:
Just worried it's there for a reason as I couldn't find the way to have it displayed... I'll leave it like that just to be safe